src/features/store/store.repository.ts

```diff
@@ -0,0 +1,82 @@
+import prisma from '../../lib/prisma';
+import { CreateStoreDto, UpdateStoreDto } from './store.dto';
+
+export class StoreRepository {
+  async findByUserId(userId: string) {
+    return prisma.store.findUnique({ where: { userId } });
+  }
+
+  async findByStoreId(storeId: string) {
+    const store = await prisma.store.findUnique({
+      where: { id: storeId },
+    });
+
+    if (!store) return null;
+
+    return store;
+  }
```

Comment:
```suggestion
  async findByStoreId(storeId: string) {
    return prisma.store.findUnique({ where: { id: storeId } });
  }
```

아마 동일한 기능을 할 거 같습니다. null체크를 굳이 하신다면 서비스로 넘기시는 게 나을 것 같네요.